modules/videoio/src/backend_plugin.cpp

```diff
@@ -534,23 +534,23 @@ class PluginWriter : public cv::IVideoWriter
     {
         CV_Assert(plugin_api);
         CvPluginWriter writer = NULL;
-        if (plugin_api->api_header.api_version >= 1 && plugin_api->Writer_open_with_params)
+        if (plugin_api->api_header.api_version >= 1 && plugin_api->v1.Writer_open_with_params)
```

Comment:
`v0` access doesn't require `api_version` checks
`v1`+ access requires `api_version` check and compatibility warnings about missing features `CV_LOG_WARNING(... due to lower API level)`